dev-resources/template/marketplace/ghactions-imagebuilder.yml

```diff
@@ -0,0 +1,39 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
so im guessing this one and the next one are more for US not for the actual client deployment?